packages/pixeloven/cli/src/commands/lint.test.ts

```diff
@@ -92,12 +92,12 @@ describe("@pixeloven/cli", () => {
                 expect(Stub.process.exit.calledOnce).toEqual(true);
                 expect(Stub.process.exit.calledWithExactly(0)).toEqual(true);
             });
-            it("should succeed linting ts/tsx with stylelint.json", async () => {
+            it("should succeed linting css/scss with stylelint.json", async () => {
                 Mock.core.expects("resolvePath").returns(true);
                 Stub.system.spawn.resolves({
                     status: 0,
                 });
-                const context = await cli.run("lint scss");
+                const context = await cli.run("lint css");
```

Comment:
Oh wait, are you reducing the task arguments to `css` and `tsx`? If we're going to do that, can we do `styles` and `scripts`?

Reply:
Negative, same as the above comment was a simple way to get more coverage without needing a new test for an alias